src/diagonal.works/b6/world.go

```diff
@@ -139,6 +121,9 @@ func InvalidTag() Tag {
 	return Tag{Value: NewStringExpression("")}
 }
 
+// TODO(mari): try to rename.
+// Technically, this interface is not taggabble, b/c it's not editable, but tagged is taken;
```

Comment:
This is exactly the thought process I went through when naming, but didn't get anywhere, so just left it as Taggable. I think Tagged is probably the best option, and we'd just have to change the existing use somehow.

Reply:
i ve tried and given up at least twice, but now i m so serious about it i m leaving a todo!